File: app/src/main/java/com/app/l_pesa/common/CustomButtonRegular.java
package com.app.l_pesa.common;

import android.content.Context;
import android.graphics.Typeface;
import android.util.AttributeSet;

public class CustomButtonRegular extends android.support.v7.widget.AppCompatButton
{
    public CustomButtonRegular(Context context) {
        super(context);
        init(context);
    }

    public CustomButtonRegular(Context context, AttributeSet attrs) {
        super(context, attrs);
        init(context);
    }

    public CustomButtonRegular(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
        init(context);
    }

    private void init(Context mContext) {
        Typeface face = Typeface.createFromAsset(mContext.getAssets(), "fonts/Montserrat-Regular.ttf");
        this.setTypeface(face);

    }
}

File: app/src/main/java/com/app/l_pesa/common/CustomButtonRegular.kt
package com.app.l_pesa.common

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet

class CustomButtonRegular : androidx.appcompat.widget.AppCompatButton {
    constructor(context: Context) : super(context) {
        init(context)
    }

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
        init(context)
    }

    constructor(context: Context, attrs: AttributeSet, defStyle: Int) : super(context, attrs, defStyle) {
        init(context)
    }

    private fun init(mContext: Context) {
        val face = Typeface.createFromAsset(mContext.assets, "fonts/Montserrat-Regular.ttf")
        this.typeface = face

    }
}

